### ISSUE TITLE
No trivial way to use pos_to_linecol within object processor

### ISSUE THREAD

**Issue by wryun** [textX/textX#189]
If you register an object processor, there appears to be no obvious way you can find out the line/column number to report a failure, because this relies on the parser which is only available at the top-level (and if you throw an exception in the object processor, the top level model won't be returned...). This would be particularly useful when trying to execute a language as it is parsed.

Personally, I think that the current pos_to_linecol interface is a little odd (two 'private' _ variables required!), and it would be very useful if all models had an attached method which simpler returned the line/column (e.g. get_linecol() or similar).

**Comment by igordejanovic** [comment]
Yes, you are definitely right. Getting line/col is not very intuitive. One would have to do something like `get_model(obj)._tx_parser.pos_to_linecol(obj._tx_position)` which is not either easy nor readable.

The main idea of having an underscore `_tx_` attributes is to prevent name-space pollution of the domain model which might be provided by the user so having `get_linecol()` directly on model class probably is not something that we should do as it would be too intrusive. But, we could extend the [Model API](http://textx.github.io/textX/latest/model/#model-api) with a function `get_linecol(object)` which will be a shortcut for the above expression.
